File: python/nivel 3 listas textos/20.py
"""
Ejercicio 20
Buscar un número en una lista (búsqueda manual)
"""

numeros = [4,15,2,89,23,7]

buscar = int(input("Ingrese el numero a buscar: "))

encontrado = False

for numero in numeros:
  if numero == buscar:
    encontrado = True
    break

if encontrado:
  print("El numero se encuentra en la lista")
else:
  print("El numero no se encuentra en la lista")

File: javascript/nivel 3 listas textos/20.js
/*
Ejercicio 20
Buscar un número en una lista (búsqueda manual)
*/

let numeros = [4,15,2,89,23,7];

let buscar = parseInt(prompt("Ingrese el numero a buscar: "));

let encontrado = false;

for (let i = 0; i < numeros.length; i++) {
  if(numeros[i] == buscar){
    encontrado = true;
    break;
  }
}

if(encontrado){
  console.log("Numero Encontrado");
}else{
  console.log("Numero No Encontrado");
}